frontend/src/metabase/visualizations/echarts/pie/model/index.unit.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { color } from "metabase/lib/colors";
+
+import { DEFAULT_SETTINGS, MOCK_RAW_SERIES } from "../test";
+import { getPieChartModel } from ".";
+
+const MOCK_RENDERING_CONTEXT = {
+  getColor: color,
+  measureText: () => 0,
+  formatValue: (_: any, _2: any) => "",
+  fontFamily: "",
+};
+
+describe("getPieChartModel", () => {
+  it("should return the correct model object given valid 'rawSeries' and 'settings'", () => {
```

Comment:
For now I'm just gonna add this basic test case as a sanity check. I am keeping track of more cases to test here and in other places, and if I have time later I'll try to add those as well.